### DISCUSSION TITLE
Reset password 24 hours

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#16622] [Archive]
When resetting a password triggered by the user, 24 hours is a reasonable duration.

But I use this feature to invite users. (May be there is a better way?)

1. I create by hand the user
2. I disconnect and act as forgot password, entering the user mail to trigger the sending of the mail

It happens that users will not necessarily click on the reset password link within 24 hours.

Is there a way to modify (even hard coding) the 24 hours duration?
or
Is the a better way to invite users?

--

A side question, as we are on password resetting:
Is it possible to have [a regex](https://www.ocpsoft.org/tutorials/regular-expressions/password-regular-expression/) we may modify to accept only more secure passwords

**Comment by br41nslug** [comment]
This 24h timeout is hardcoded unfortunately and cannot be edited without forking directus currently :( https://github.com/directus/directus/blob/main/api/src/services/users.ts#L386

The password security you can manage from the app https://docs.directus.io/configuration/project-settings.html#security
